rswag-specs/lib/rswag/specs/example_group_helpers.rb

```diff
@@ -132,7 +138,7 @@ def run_test!(&block)
           end
 
           it "returns a #{metadata[:response][:code]} response", rswag: true do |example|
-            assert_response_matches_metadata(example.metadata, &block)
+            assert_response_matches_metadata(example.metadata.merge(options), &block)
```

Comment:
Instead of merging options with metadata, could we pass it to `it`?

```ruby
      def run_test!(**options, &block)
        options[:rswag] = true unless options.key?(:rswag)

        if RSPEC_VERSION < 3
          ActiveSupport::Deprecation.warn('Rswag::Specs: WARNING: Support for RSpec 2.X will be dropped in v3.0')
          before do
            submit_request(example.metadata)
          end

          it "returns a #{metadata[:response][:code]} response", **options do
            assert_response_matches_metadata(metadata)
            block.call(response) if block_given?
          end
        else
          before do |example|
            submit_request(example.metadata)
          end

          it "returns a #{metadata[:response][:code]} response", **options do |example|
            assert_response_matches_metadata(example.metadata, &block)
            example.instance_exec(response, &block) if block_given?
          end
        end
      end
```

This would allow us to pass in options to `it`, which is a feature I'd like to have.  For example, this would allow us to do the following:
```ruby
run_test! focus: true
```

Reply:
Thank you!  I put up a sub-PR here if that is helpful:
https://github.com/IlkhamGaysin/rswag/pull/1

Reply:
@IlkhamGaysin, sounds good!

Reply:
@romanblanco done!